flight/Modules/Actuator/actuator.c

```diff
@@ -691,7 +589,7 @@ static bool set_channel(uint8_t mixer_channel, float value, const ActuatorSettin
                             }
 			}
 			PIOS_Servo_Set(mixer_channel,
-							buzzOn?actuatorSettings->ChannelMax[mixer_channel]:actuatorSettings->ChannelMin[mixer_channel]);
+							buzzOn?actuatorSettings.ChannelMax[mixer_channel]:actuatorSettings.ChannelMin[mixer_channel]);
```

Comment:
Not your fault in the original, but could we get some spaces here as long as that line is being changed?